staging/src/k8s.io/code-generator/cmd/validation-gen/validation.go

```diff
@@ -1462,6 +1471,82 @@ func toGolangSourceDataLiteral(sw *generator.SnippetWriter, c *generator.Context
 		sw.Do("$.|private$", c.Universe.Type(types.Name(v)))
 	case *validators.PrivateVar:
 		sw.Do("$.|private$", c.Universe.Type(types.Name(*v)))
+	case validators.WrapperFunction:
```

Comment:
Neat trick. The composition this offers is really great.

Reply:
I learned it from watching you, Joe!